delivery_purchase_multi_destination/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This module allows to set different price rules depending on the origin.
```

Comment:
You have to put here that this is the glue module for using multi destinations for purchase sendings.